omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb

```diff
@@ -52,13 +52,21 @@
     # Whitelist the docs necessary to serve up error pages and friendly
     # html to non-chef clients hitting this host.
     location ~ "^/[0-9]{3,3}\.(json|html)|favicon.ico|index.html$" {
+      add_header Content-Security-Policy "default-src 'self';";
+      add_header X-Frame-Options DENY;
```

Comment:
https://github.com/chef/chef-server/blob/298eb078aedc2a5e1a7da9408db94116ed27f9d1/omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/nginx_chef_api_lb.conf.erb#L40-L41 seems to add this option only when manage is not installed.

Reply:
Wouldn't this page not display is manage was installed?